traefik/rootfs/etc/traefik/traefik.yaml.gotmpl

```diff
@@ -20,6 +20,9 @@ entryPoints:
 api:
   dashboard: true
   insecure: true
+  
+pilot:
```

Comment:
Have you tested this? What happens if I don't provide a token? Maybe an conditional statement around this configuration block is necessary.

Reply:
No, I haven't. I saw just noticed the correct way to do it and added the if block around it. Thanks for pointing that out !